rules/dns-what-and-how-it-works/rule.md

```diff
@@ -0,0 +1,42 @@
+---
+type: rule
+title: Do you know DNS, what it is and how it works?
```

Comment:
Hi @Dhruv-0987 

What about changing to "Do you know what DNS is and how it works?"

Reply:
Hi @tiagov8 thanks for the feedback I have changed it so.